src/test/java/de/tum/in/www1/artemis/exam/ExamIntegrationTest.java

```diff
@@ -887,7 +878,7 @@ private void testAllPreAuthorize() throws Exception {
     }
 
     @Test
-    @WithMockUser(username = TEST_PREFIX + "instructor6", roles = "INSTRUCTOR")
+    @WithMockUser(username = TEST_PREFIX + "instructor10", roles = "INSTRUCTOR")
```

Comment:
Why did you increase this to 10? Here as well as a few test further down

Reply:
Before there were two instructors (instructor6 and instructor10), that got added in the init() method and did not belong to course1. In the few test cases where instructor6 was used, I noticed that instructor10 could also be used instead since the tests only need an instructor who is not in course1. This is why I removed instructor6 entirely.